src/mac.js

```diff
@@ -189,21 +189,31 @@ class MacApp extends App {
 
     const plists = await this.determinePlistFilesToUpdate()
     await Promise.all(plists.map(plistArgs => this.loadPlist(...plistArgs)))
-    await this.extendAppPlist(this.opts.extendInfo)
+    await this.extendPlist(this.appPlist, this.opts.extendInfo)
     this.appPlist = this.updatePlist(this.appPlist, this.executableName, appBundleIdentifier, this.appName)
-    this.helperPlist = this.updateHelperPlist(this.helperPlist)
+
     const updateIfExists = [
       ['helperRendererPlist', '(Renderer)', true],
       ['helperPluginPlist', '(Plugin)', true],
       ['helperGPUPlist', '(GPU)', true],
       ['helperEHPlist', 'EH'],
       ['helperNPPlist', 'NP']
     ]
+
+    for (const [plistKey] of [...updateIfExists, ['helperPlist']]) {
```

Comment:
The first arg of each array in `updateIfExists` is used as a key. Maybe this variable should be an object instead of an array of arrays e.g.

```
    const updateIfExists = { 
      'helperRendererPlist': ['(Renderer)', true],
      'helperPluginPlist': ['(Plugin)', true],
      'helperGPUPlist': ['(GPU)', true],
      'helperEHPlist': ['EH'],
      'helperNPPlist': ['NP']
    }
```

Not a big deal but it would make the code that uses it a little easier to read

Reply:
It's used as a key on a different object, I think this would be harder to read because you'd have to pull the keys and values separately via `entries()` or with two calls two `keys()` and `values()`.  Also code hasn't been introduced here, in the spirit of minimal and related changes we probably shouldn't refactor here.